src/header.rs

```diff
@@ -54,56 +51,29 @@ struct TbfHeaderWriteableFlashRegion {
 
 impl fmt::Display for TbfHeaderBase {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(
-            f,
-            "
-               version: {:>8} {:>#10X}
-           header_size: {:>8} {:>#10X}
-            total_size: {:>8} {:>#10X}
-                 flags: {:>8} {:>#10X}
-",
-            self.version,
-            self.version,
-            self.header_size,
-            self.header_size,
-            self.total_size,
-            self.total_size,
-            self.flags,
-            self.flags,
-        )
+        writeln!(f)?;
+        writeln!(f, "version: {:>8} {:>#10X}", self.version,self.version)?;
+        writeln!(f, "header_size: {:>8} {:>#10X}", self.header_size, self.header_size)?;
+        writeln!(f, "total_size: {:>8} {:>#10X}", self.total_size, self.total_size)?;
+        writeln!(f, "flags: {:>8} {:>#10X}", self.flags, self.flags)
```

Comment:
This removes all of the alignment that Pat added to make the printout easier to read.

Reply:
Ok, I can revert this! I guess this applies to all `write! stuff`?